delivery_dropoff_site/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,35 @@
+This module extend Odoo functionnalities, regarding delivery features to
+add a new concept of drop-off Sites.
+
+Main international carriers provide transportation services to specific areas
+managed by them or by subcontractors.
+
+Then, recipients come pick up their packages in these sites.
+
+Usage
+=====
```

Comment:
Why not use USAGE.rst file ?

It could be improve presentation polymorphism : a la odoo apps, etc.